boxoffice/views/admin.py

```diff
@@ -26,9 +27,11 @@ def index():
 
 def jsonify_org(data):
     item_collections_list = ItemCollection.query.filter(ItemCollection.organization == data['org']).order_by('created_at desc').all()
+    html_form = render_form(form=ItemCollectionForm(), title=u"New Item Collection", submit=u"Save", ajax=False, with_chrome=False)
```

Comment:
`ajax` is `False` by default in `render_form` so it isn't required as an argument.